rebar.config.script

```diff
@@ -32,7 +32,15 @@ CouchJSName = case os:type() of
         "couchjs"
 end,
 CouchJSPath = filename:join(["priv", CouchJSName]),
-Version = string:strip(os:cmd("git describe --always"), right, $\n),
+Version0 = string:strip(os:cmd("git describe --always"), right, $\n),
+% When we are not building from a git checkout, the above will fail.
+% If that is the case, we use the COUCHDB_VERSION variable that has
+% been set by the top level Makefile
+Version = case Version0 of
+    "fatal:" ++ _Msg ->
```

Comment:
Hm...but how we'll transfer COUCHDB_VERSION value to the tarball in this case? I'd tried to solve that problem, but everytime it ended that I had to create some version file on make dist step and read the version from there. Better was some pregenerated rebar.conf.script, but it turns to be more complicated as I hoped.